src/test/java/RequestKeyCheckInterceptorTest.java

```diff
@@ -0,0 +1,152 @@
+import com.google.inject.Inject;
+import filemanager.exceptions.DateIsEmptyException;
+import filemanager.interceptor.RequestKeyCheckInterceptor;
+import filemanager.model.Command;
+import org.aopalliance.intercept.MethodInvocation;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.ws.rs.ForbiddenException;
+import java.lang.reflect.AccessibleObject;
+import java.lang.reflect.Method;
+import java.time.LocalDateTime;
+
+import static org.junit.Assert.assertEquals;
+
+public class RequestKeyCheckInterceptorTest {
+
+    @Inject
+    RequestKeyCheckInterceptor requestKeyCheckInterceptor;
+
+    @After
+    public void tearDown() {
+        requestKeyCheckInterceptor = null;
+    }
+
+    @Before
+    public void setUp() {
+        requestKeyCheckInterceptor = new RequestKeyCheckInterceptor();
```

Comment:
That's ok implementation. But it has too much knowledge of our app. I'd prefer you **not** to test the interceptor directly, but to work with our service from User perspective. For this you need to start your tests with spinning up your service and using some HttpClient to send requests to it. Here is some related article that may help you with spinning up your application for the tests without lots of boilerplate code: https://www.dropwizard.io/1.2.2/docs/manual/testing.html#integration-testing